.gitignore

```diff
@@ -14,3 +16,5 @@ unit/coverage/*.gcov
 /daemon/test/sailjail/permissions/
 /report/
 /coverage
+documentation.list
```

Comment:
Maybe just `*.list`? This will cover `mb2 build -d` case as well.

Reply:
Sounds like a good idea.